packages/sanity/src/core/releases/tool/detail/documentTable/useReleaseHistory.ts

```diff
@@ -21,14 +24,14 @@ export function useReleaseHistory(bundleDocumentsIds: string[]): {
   const {dataset, token} = client.config()
   const [history, setHistory] = useState<TransactionLogEventWithEffects[]>([])
   const queryParams = `tag=sanity.studio.tasks.history&effectFormat=mendoza&excludeContent=true&includeIdentifiedDocumentsOnly=true`
-
-  const publishedIds = bundleDocumentsIds.map((id) => getPublishedId(id)).join(',')
+  const publishedIds = bundleDocumentsIds.map((id) => getVersionId(id, releaseId)).join(',')
```

Comment:
I think we might need to rename the const to documentIds or maybe add a comment to clarify? Just bringing it up since we are technically bringing up a bunch of version ids (which are not publishedIds) 🤔

Reply:
I agree @RitaDias. Based on my understanding from glancing over the code, at least 🙂.

Reply:
TRUE missed that rename! will push the update and merge